js/utils/getCmdLineArg.js

```diff
@@ -9,17 +9,20 @@ const logLevels = require('../enums/logLevels.js');
  * @param  {String} argName   Arg name to search for.
  * @param  {Boolean} exactMatch  If true then look for exact match otherwise
  * try finding arg that starts with argName.
+ * @param {Boolean} caseSensitive Optional. If set to false, the comparison is not case sensitive
  * @return {String}           If found, returns the arg, otherwise null.
  */
-function getCmdLineArg(argv, argName, exactMatch) {
+function getCmdLineArg(argv, argName, exactMatch, caseSensitive=true) {
```

Comment:
Alternatively, we don't have an arg and simply always do cases insensitive comparison. Either way works for me